Generator/Commands/ContainerGenerator.php

```diff
@@ -192,6 +192,82 @@ public function getUserInputs()
                 'task'      => 'Delete' . $model . 'Task',
             ],
         ];
+
+        if ($ui == 'web') {
+            $routes = [
```

Comment:
This really looks weired.. Because if we change something on one array-field, we must change it in several locations (for the `API` and `WEB` part..
So why not just "add" the missing `array-fields` in the `if($ui == 'web')` part instead of re-declaring the variable? Would be a) more readable and b) easier to maintain!

Reply:
This is create request for API:

```
            [
                'stub'      => 'Create',
                'name'      => 'Create' . $model,
                'operation' => 'create' . $model,
                'verb'      => 'POST',
                'url'       => $url,
                'action'    => 'Create' . $model . 'Action',
                'request'   => 'Create' . $model . 'Request',
                'task'      => 'Create' . $model . 'Task',
            ],
```

This is create for WEB:

```
                [
                    'stub'      => 'Create',
                    'name'      => 'Store' . $model,
                    'operation' => 'store',
                    'verb'      => 'POST',
                    'url'       => $url . '/store',
                    'action'    => 'Create' . $model . 'Action',
                    'request'   => 'Store' . $model . 'Request',
                    'task'      => 'Create' . $model . 'Task',
                ],
```

The thing is that we need to have the same Action and Task for store on the WEB UI. We can not create a CreateResourceRequest and have it in the store method, what request then should the create method have(the one to show the creation form)?

How do you think we should handle this?

Because using the same fields for both API and WEB is not going to work.

Reply:
ok.. I admit I only develop APIs, so i don't know that much about, how "native WEB" applications (with pure HTML / BLADE) are built in laravel or what their specific needs are...

As it looks like, there are quite some differences - how about we just make a dedicated `php artisan apiato:generate:container:web` command and a a dedicated `php artisan apiato:generate:container:api` command? This way, the generation process can be more easily customized without interfering with already existing logic..

Furthermore, we just use the already existing `php artisan apiato:generate:container` to call (one or both) the others..

What do you think?

Reply:
Agree. That would make everything better.

Reply:
I think, it would be much more easy to read / understand.. Furthermore, we could easily adapt the `ContainerWebGenerator` to specific needs (e.g., we do not need a `Transformer`), so simply don't call it, instead of wrapping it in a `if/else block`..

Reply:
Can you do the refactoring to a dedicated `ContainerWebGenerator`? Actually i think, you now more about the `WEB` part than i do (or what is needed specifically, ...)

Reply:
Sure, I'll get to it ;)